packages/graph-rag-example-helpers/src/graph_rag_example_helpers/utils.py

```diff
@@ -0,0 +1,3 @@
+def add_tabs(text: str) -> str:
```

Comment:
I believe this is just `textwrap.indent("\t", text)`.

https://docs.python.org/3/library/textwrap.html#textwrap.indent